src/test/resources/spec/concordion/command/example/Example.html

```diff
@@ -93,14 +93,66 @@
 
 </div>
 
+
+<div class="example">
+    <h3>Tables</h3>
+
+    <p>
+        Concordion can execute each row in a table as an example. To do this, use the c:exampleName tag in one of the table header rows.
+    </p>
+
+    <p>For example:</p>
+
+    <code><pre>
+    &lt;table c:execute="#counter = incrementCounter()"&gt;
+        &lt;tr&gt;
+            &lt;th c:exampleName="#TEXT"&gt;Example Name&lt;/th&gt;
+            &lt;th c:assertEquals="#counter"&gt;Counter value&lt;/th&gt;
+        &lt;/tr&gt;
+        &lt;tr&gt;
+            &lt;td&gt;1st Table Example&lt;/td&gt;
+            &lt;td&gt;11&lt;/td&gt;
+        &lt;/tr&gt;
+        &lt;tr&gt;
+            &lt;td&gt;2nd Table Example&lt;/td&gt;
+            &lt;td&gt;11&lt;/td&gt;
+        &lt;/tr&gt;
+        &lt;tr&gt;
+            &lt;td&gt;3rd Table Example&lt;/td&gt;
+            &lt;td&gt;11&lt;/td&gt;
+        &lt;/tr&gt;
+    &lt;/table&gt;
+</pre></code>
+
+
+    <table c:execute="#counter = incrementCounter()">
+        <tr>
+            <th c:exampleName="#TEXT">Example Name</th>
+            <th c:assertEquals="#counter">Counter value</th>
+        </tr>
+        <tr>
+            <td>1st Table Example</td>
+            <td>11</td>
+        </tr>
+        <tr>
+            <td>2nd Table Example</td>
+            <td>11</td>
+        </tr>
+        <tr>
+            <td>3rd Table Example</td>
+            <td>11</td>
+        </tr>
+    </table>
+</div>
+
```

Comment:
Suggest we think about how to improve the spec so that it is checking we get individual examples with the correct  names